pkg/cmd/repo/delete/delete.go

```diff
@@ -60,8 +66,15 @@ func deleteRun(opts *DeleteOptions) error {
 	}
 	apiClient := api.NewClientFromHTTP(httpClient)
 
-	repoSelector := opts.RepoArg
 	var toDelete ghrepo.Interface
+	repoSelector := opts.RepoArg
+	if repoSelector == "" {
+		currRepo, err := opts.BaseRepo()
+		if err != nil {
+			return err
+		}
+		repoSelector = ghrepo.FullName(currRepo)
+	}
 
 	if !strings.Contains(repoSelector, "/") {
```

Comment:
Seems like this could be simplified a bit.
```go
	var toDelete ghrepo.Interface

	if opts.RepoArg == "" {
		toDelete, err = opts.BaseRepo()
		if err != nil {
			return err
		}
	} else {
		repoSelector := opts.RepoArg
		if !strings.Contains(repoSelector, "/") {
			currentUser, err := api.CurrentLoginName(apiClient, ghinstance.Default())
			if err != nil {
				return err
			}
			repoSelector = currentUser + "/" + repoSelector
		}
		toDelete, err = ghrepo.FromFullName(repoSelector)
		if err != nil {
			return fmt.Errorf("argument error: %w", err)
		}
	}
```